sdks/js/packages/core/react/components/organization/billing/index.tsx

```diff
@@ -122,7 +122,7 @@ const PaymentMethod = ({
     <div className={billingStyles.detailsBox}>
       <Flex align={'center'} justify={'between'} style={{ width: '100%' }}>
         <Text className={billingStyles.detailsBoxHeading}>Payment method</Text>
-        <Button variant={'secondary'}>Add method</Button>
+        {/* <Button variant={'secondary'}>Add method</Button> */}
```

Comment:
remove if not needed

Reply:
the button is in design but the functionality is not built in backend yet, thats why keeping it commented now